starboard/shared/starboard/player/decoded_audio_internal.cc

```diff
@@ -68,9 +68,25 @@ DecodedAudio::DecodedAudio(int channels,
       size_in_bytes_(size_in_bytes) {
   SB_DCHECK(channels_ > 0);
   SB_DCHECK(size_in_bytes_ >= 0);
-  // TODO(b/275199195): Enable the SB_DCHECK below.
-  // SB_DCHECK(size_in_bytes_ % (GetBytesPerSample(sample_type_) * channels_) ==
-  //           0);
+}
```

Comment:
The SB_DCHECK() should be here too.  If it's triggered by some code, we should either refine these code, or keep the TODO here.